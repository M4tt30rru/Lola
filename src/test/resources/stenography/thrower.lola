##import "std.lola"
##Find
 ##MethodDeclaration(_dec) = throws ##Any(_value);
 ##replace ##(_dec) {throw new ##(_value);}

void error(String message) = throws BadError(message);
