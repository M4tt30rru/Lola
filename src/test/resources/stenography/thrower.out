    void  error ( Stringmessage ) {throw new BadError(message);}
